app/helpers/hosts_helper.rb

```diff
@@ -113,7 +113,8 @@ def name_column(record)
   end
 
   def days_ago(time)
-    ((Time.zone.now - time) / 1.day).round.to_i
+    ago = ((Time.zone.now - time) / 1.day).round.to_i
```

Comment:
Should .round be .ceil instead?

The reason I'm thinking this is for a case where you have two reports in Foreman, one from 26 hours ago, and one from 2 hours ago.  days_ago(3.hours.ago) would still return 1 (this is what you've fixed), while days_ago(26.hours.ago) would also then return 2, so you'd be able to filter on reports in the last two days (48 hours).